papery/page.py

```diff
@@ -72,7 +72,7 @@ def text(self, lang=None, link=None):
                                              # TODO make "image_path" configurable by papery's configuration file for security reasons
                                          }},
                                      "toc": {
-                                         "marker": "[toc]",
+                                         "marker": "[TOC]",
```

Comment:
The `marker`'s default is `[TOC]`, so we're able to apply it by only deleting this config.
cf. https://python-markdown.github.io/extensions/toc/#usage
```suggestion
```